setup.py

```diff
@@ -32,15 +30,14 @@
       keywords     = 'Robot Framework',
       platforms    = 'any',
       install_requires= [
-          "robotframework==3.0",
-          "robotframework-requests==0.4.5",
-          "robotframework-selenium2library==1.7.4",
+          "robotframework==3.0.2",
+          "robotframework-requests==0.4.7",
+          "robotframework-selenium2library==3.0.0",
```

Comment:
I don't think we should be pulling Selenium2Library anymore. Don't we want to switch to the SeleniumLibrary instead?

https://github.com/robotframework/SeleniumLibrary#versions